src/content/learn/tutorials/general/releases.md

```diff
@@ -37,11 +37,15 @@ For example, a release for version 1.0.0 would be tagged `v1.0.0`.
 To test a specific release, [clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) and checkout the release's associated tag.
 Be sure to [clone the repository with submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules#_cloning_submodules).
 
-As an example, to test the [OSCAL 1.0.5 pre-release](https://github.com/usnistgov/OSCAL/releases/tag/v1.0.5), you could clone the repository directly at the release tag:
+As an example, to test the [OSCAL 1.1.0 pre-release](https://github.com/usnistgov/OSCAL/releases/tag/v1.1.0), you could clone the repository directly at the release tag:
+
+{{% callout note %}}Checking out a repository on a tag may yield a warning about being in a "detached HEAD" state, for more details, see the [Git documentation](https://git-scm.com/docs/git-checkout#_detached_head){{% /callout %}}
 
 ```sh
-# Clone the OSCAL repository with submodules at the 1.0.5 release
-git clone --recurse-submodules https://github.com/usnistgov/OSCAL.git --branch v1.0.5
+# Clone the OSCAL repository with submodules at the 1.1.0 release
+# Note: this will create a sparse commit, and Git will warn you about being in a "detached head" state.
+# Alternatively clone the repository wi
```

Comment:
Minor (but iffy): we should finish the thought here.

```suggestion
# Alternatively clone the repository with the default branch, and subsequently checkout a branch with the `release-` prefix or tag accordingly.
```

Reply:
Good catch. I forgot to remove this line but it's good to have it in the callout and here in a code excerpt as well.